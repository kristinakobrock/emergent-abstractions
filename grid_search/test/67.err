  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 18.48it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 24.31it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 26.21it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 28.85it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 27.58it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 27.20it/s] 46%|##########################################3                                                 | 23/50 [00:00<00:01, 24.55it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 28.06it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 28.56it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 28.87it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 29.10it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 26.58it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 24.88it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 23.86it/s] 96%|########################################################################################3   | 48/50 [00:01<00:00, 23.27it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.75it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.20it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 27.66it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 24.88it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 23.61it/s]100%|############################################################################################| 13/13 [00:00<00:00, 23.89it/s]
