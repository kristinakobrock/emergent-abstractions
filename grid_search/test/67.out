# ["--attributes=5", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7031366229057312, "acc": 0.5788488984107971, "length": 3.503866672515869, "mode": "train", "epoch": 1}
{"loss": 0.6978199481964111, "acc": 0.5056641101837158, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6681331396102905, "acc": 0.627350389957428, "length": 3.3459177017211914, "mode": "train", "epoch": 2}
{"loss": 0.6877137422561646, "acc": 0.5582031607627869, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.661744236946106, "acc": 0.6290776133537292, "length": 3.1076459884643555, "mode": "train", "epoch": 3}
{"loss": 0.6751184463500977, "acc": 0.6123046875, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6603456139564514, "acc": 0.633391797542572, "length": 2.845114231109619, "mode": "train", "epoch": 4}
{"loss": 0.6749187111854553, "acc": 0.6142578721046448, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.657840371131897, "acc": 0.6331687569618225, "length": 2.776257276535034, "mode": "train", "epoch": 5}
{"loss": 0.6662708520889282, "acc": 0.6226562857627869, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6569344401359558, "acc": 0.633421778678894, "length": 2.783827066421509, "mode": "train", "epoch": 6}
{"loss": 0.6637896299362183, "acc": 0.6242188811302185, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6551701426506042, "acc": 0.6329646706581116, "length": 2.6143250465393066, "mode": "train", "epoch": 7}
{"loss": 0.6635136604309082, "acc": 0.6242188811302185, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6520881056785583, "acc": 0.6333965063095093, "length": 2.4347219467163086, "mode": "train", "epoch": 8}
{"loss": 0.6590515375137329, "acc": 0.628987193107605, "length": 1.2109375, "mode": "test", "epoch": 8}
{"loss": 0.6424797177314758, "acc": 0.6377527713775635, "length": 2.5547077655792236, "mode": "train", "epoch": 9}
{"loss": 0.6448743939399719, "acc": 0.6390396356582642, "length": 1.76171875, "mode": "test", "epoch": 9}
{"loss": 0.6293404698371887, "acc": 0.6537469029426575, "length": 2.6225666999816895, "mode": "train", "epoch": 10}
{"loss": 0.6323741674423218, "acc": 0.6490906476974487, "length": 2.01953125, "mode": "test", "epoch": 10}
{"loss": 0.6168964505195618, "acc": 0.6676448583602905, "length": 2.645555019378662, "mode": "train", "epoch": 11}
{"loss": 0.6261428594589233, "acc": 0.6511536240577698, "length": 2.16015625, "mode": "test", "epoch": 11}
{"loss": 0.6006587743759155, "acc": 0.6795928478240967, "length": 2.75516676902771, "mode": "train", "epoch": 12}
{"loss": 0.6002994775772095, "acc": 0.6676666736602783, "length": 3.3046875, "mode": "test", "epoch": 12}
{"loss": 0.5787353515625, "acc": 0.6987289190292358, "length": 3.1872317790985107, "mode": "train", "epoch": 13}
{"loss": 0.595818281173706, "acc": 0.6877105236053467, "length": 3.8828125, "mode": "test", "epoch": 13}
{"loss": 0.5562385320663452, "acc": 0.7176849842071533, "length": 3.498908519744873, "mode": "train", "epoch": 14}
{"loss": 0.5920314788818359, "acc": 0.7031433582305908, "length": 3.8828125, "mode": "test", "epoch": 14}
{"loss": 0.5398937463760376, "acc": 0.7263641357421875, "length": 3.4911649227142334, "mode": "train", "epoch": 15}
{"loss": 0.5828112363815308, "acc": 0.7064880728721619, "length": 3.734375, "mode": "test", "epoch": 15}
{"loss": 0.5271557569503784, "acc": 0.7357203960418701, "length": 3.5117132663726807, "mode": "train", "epoch": 16}
{"loss": 0.5859293937683105, "acc": 0.7200545072555542, "length": 3.54296875, "mode": "test", "epoch": 16}
{"loss": 0.5122918486595154, "acc": 0.7466607689857483, "length": 3.4493401050567627, "mode": "train", "epoch": 17}
{"loss": 0.5832998752593994, "acc": 0.7233322858810425, "length": 3.70703125, "mode": "test", "epoch": 17}
{"loss": 0.5019757151603699, "acc": 0.7536277174949646, "length": 3.5580832958221436, "mode": "train", "epoch": 18}
{"loss": 0.579314649105072, "acc": 0.7282043695449829, "length": 3.66796875, "mode": "test", "epoch": 18}
{"loss": 0.4868585765361786, "acc": 0.7615261077880859, "length": 3.6033918857574463, "mode": "train", "epoch": 19}
{"loss": 0.5671586990356445, "acc": 0.7277342677116394, "length": 3.73046875, "mode": "test", "epoch": 19}
{"loss": 0.47481319308280945, "acc": 0.7726590037345886, "length": 3.5696539878845215, "mode": "train", "epoch": 20}
{"loss": 0.5695105791091919, "acc": 0.7389022707939148, "length": 3.77734375, "mode": "test", "epoch": 20}
{"loss": 0.4605199694633484, "acc": 0.7842727899551392, "length": 3.6698968410491943, "mode": "train", "epoch": 21}
{"loss": 0.5499957203865051, "acc": 0.7533736228942871, "length": 3.765625, "mode": "test", "epoch": 21}
{"loss": 0.44408413767814636, "acc": 0.7957239151000977, "length": 3.6830739974975586, "mode": "train", "epoch": 22}
{"loss": 0.5371303558349609, "acc": 0.763720691204071, "length": 3.734375, "mode": "test", "epoch": 22}
{"loss": 0.4313313066959381, "acc": 0.8056644797325134, "length": 3.6323771476745605, "mode": "train", "epoch": 23}
{"loss": 0.5331207513809204, "acc": 0.7633102536201477, "length": 3.82421875, "mode": "test", "epoch": 23}
{"loss": 0.42069143056869507, "acc": 0.8103570938110352, "length": 3.724093198776245, "mode": "train", "epoch": 24}
{"loss": 0.5319141149520874, "acc": 0.76300048828125, "length": 3.77734375, "mode": "test", "epoch": 24}
{"loss": 0.4061487317085266, "acc": 0.8196414113044739, "length": 3.7040855884552, "mode": "train", "epoch": 25}
{"loss": 0.5431447625160217, "acc": 0.7630279064178467, "length": 3.73046875, "mode": "test", "epoch": 25}
{"loss": 0.39808735251426697, "acc": 0.825112521648407, "length": 3.7495384216308594, "mode": "train", "epoch": 26}
{"loss": 0.5301672220230103, "acc": 0.7554275393486023, "length": 3.8671875, "mode": "test", "epoch": 26}
{"loss": 0.3894062936306, "acc": 0.8322649598121643, "length": 3.7440598011016846, "mode": "train", "epoch": 27}
{"loss": 0.5237957239151001, "acc": 0.7585784196853638, "length": 3.77734375, "mode": "test", "epoch": 27}
{"loss": 0.3811652958393097, "acc": 0.8332153558731079, "length": 3.762096405029297, "mode": "train", "epoch": 28}
{"loss": 0.5180118083953857, "acc": 0.7702072262763977, "length": 3.83984375, "mode": "test", "epoch": 28}
{"loss": 0.3716796934604645, "acc": 0.8412126898765564, "length": 3.79630446434021, "mode": "train", "epoch": 29}
{"loss": 0.5033091306686401, "acc": 0.7778106927871704, "length": 3.83203125, "mode": "test", "epoch": 29}
{"loss": 0.3658044636249542, "acc": 0.84599369764328, "length": 3.819345712661743, "mode": "train", "epoch": 30}
{"loss": 0.4905967712402344, "acc": 0.7809509634971619, "length": 3.81640625, "mode": "test", "epoch": 30}
{"loss": 0.3594636917114258, "acc": 0.8466657996177673, "length": 3.802359104156494, "mode": "train", "epoch": 31}
{"loss": 0.49504274129867554, "acc": 0.7913223505020142, "length": 3.77734375, "mode": "test", "epoch": 31}
{"loss": 0.3511974811553955, "acc": 0.8517298698425293, "length": 3.8241069316864014, "mode": "train", "epoch": 32}
{"loss": 0.48118728399276733, "acc": 0.7980453968048096, "length": 3.81640625, "mode": "test", "epoch": 32}
{"loss": 0.34721970558166504, "acc": 0.8557851910591125, "length": 3.817657709121704, "mode": "train", "epoch": 33}
{"loss": 0.4837180972099304, "acc": 0.7874801158905029, "length": 3.84375, "mode": "test", "epoch": 33}
{"loss": 0.3369028866291046, "acc": 0.8586005568504333, "length": 3.82127046585083, "mode": "train", "epoch": 34}
{"loss": 0.44897398352622986, "acc": 0.8047715425491333, "length": 3.8125, "mode": "test", "epoch": 34}
{"loss": 0.3253011107444763, "acc": 0.8649469614028931, "length": 3.8447368144989014, "mode": "train", "epoch": 35}
{"loss": 0.4604349136352539, "acc": 0.8024612665176392, "length": 3.84765625, "mode": "test", "epoch": 35}
{"loss": 0.32632699608802795, "acc": 0.8648793697357178, "length": 3.8599274158477783, "mode": "train", "epoch": 36}
{"loss": 0.4449096918106079, "acc": 0.8121855854988098, "length": 3.8203125, "mode": "test", "epoch": 36}
{"loss": 0.31901249289512634, "acc": 0.8670978546142578, "length": 3.868419885635376, "mode": "train", "epoch": 37}
{"loss": 0.4461027979850769, "acc": 0.8101822137832642, "length": 3.89453125, "mode": "test", "epoch": 37}
{"loss": 0.3112228810787201, "acc": 0.8737435936927795, "length": 3.8728420734405518, "mode": "train", "epoch": 38}
{"loss": 0.4387109875679016, "acc": 0.818219006061554, "length": 3.859375, "mode": "test", "epoch": 38}
{"loss": 0.3084191381931305, "acc": 0.8737102150917053, "length": 3.877196788787842, "mode": "train", "epoch": 39}
{"loss": 0.43726804852485657, "acc": 0.8232666254043579, "length": 3.89453125, "mode": "test", "epoch": 39}
{"loss": 0.3005977272987366, "acc": 0.8790377974510193, "length": 3.897637367248535, "mode": "train", "epoch": 40}
{"loss": 0.4519137144088745, "acc": 0.8220489025115967, "length": 3.92578125, "mode": "test", "epoch": 40}
{"loss": 0.29102128744125366, "acc": 0.8818303942680359, "length": 3.9072020053863525, "mode": "train", "epoch": 41}
{"loss": 0.43172162771224976, "acc": 0.8264617323875427, "length": 3.90234375, "mode": "test", "epoch": 41}
{"loss": 0.28684255480766296, "acc": 0.8847983479499817, "length": 3.9064834117889404, "mode": "train", "epoch": 42}
{"loss": 0.43247470259666443, "acc": 0.8308928608894348, "length": 3.8984375, "mode": "test", "epoch": 42}
{"loss": 0.2799595296382904, "acc": 0.8895256519317627, "length": 3.9186248779296875, "mode": "train", "epoch": 43}
{"loss": 0.4408702850341797, "acc": 0.8371154069900513, "length": 3.921875, "mode": "test", "epoch": 43}
{"loss": 0.28023555874824524, "acc": 0.8874280452728271, "length": 3.921297073364258, "mode": "train", "epoch": 44}
{"loss": 0.4421347379684448, "acc": 0.8320327997207642, "length": 3.9296875, "mode": "test", "epoch": 44}
{"loss": 0.2746632695198059, "acc": 0.8914252519607544, "length": 3.9215149879455566, "mode": "train", "epoch": 45}
{"loss": 0.42301517724990845, "acc": 0.8414871096611023, "length": 3.92578125, "mode": "test", "epoch": 45}
{"loss": 0.26918429136276245, "acc": 0.8940128684043884, "length": 3.9252610206604004, "mode": "train", "epoch": 46}
{"loss": 0.41019463539123535, "acc": 0.8446928858757019, "length": 3.91796875, "mode": "test", "epoch": 46}
{"loss": 0.2698480784893036, "acc": 0.893913745880127, "length": 3.9277515411376953, "mode": "train", "epoch": 47}
{"loss": 0.40345874428749084, "acc": 0.8533845543861389, "length": 3.921875, "mode": "test", "epoch": 47}
{"loss": 0.2635788917541504, "acc": 0.8974279165267944, "length": 3.944430112838745, "mode": "train", "epoch": 48}
{"loss": 0.4289224445819855, "acc": 0.849082887172699, "length": 3.92578125, "mode": "test", "epoch": 48}
{"loss": 0.2570292055606842, "acc": 0.9023943543434143, "length": 3.9364614486694336, "mode": "train", "epoch": 49}
{"loss": 0.4066752791404724, "acc": 0.8429198861122131, "length": 3.9375, "mode": "test", "epoch": 49}
{"loss": 0.24666555225849152, "acc": 0.9035854339599609, "length": 3.942945957183838, "mode": "train", "epoch": 50}
{"loss": 0.39290279150009155, "acc": 0.8552643060684204, "length": 3.93359375, "mode": "test", "epoch": 50}
{"loss": 0.23866119980812073, "acc": 0.9085515737533569, "length": 3.944284677505493, "mode": "train", "epoch": 51}
{"loss": 0.387351393699646, "acc": 0.8576720952987671, "length": 3.9375, "mode": "test", "epoch": 51}
{"loss": 0.23918624222278595, "acc": 0.9099036455154419, "length": 3.945093870162964, "mode": "train", "epoch": 52}
{"loss": 0.3896951973438263, "acc": 0.8605682253837585, "length": 3.9375, "mode": "test", "epoch": 52}
{"loss": 0.23309330642223358, "acc": 0.909972608089447, "length": 3.9448049068450928, "mode": "train", "epoch": 53}
{"loss": 0.4005383551120758, "acc": 0.851554811000824, "length": 3.953125, "mode": "test", "epoch": 53}
{"loss": 0.22928066551685333, "acc": 0.9110430479049683, "length": 3.9642529487609863, "mode": "train", "epoch": 54}
{"loss": 0.37683337926864624, "acc": 0.8655577301979065, "length": 3.96484375, "mode": "test", "epoch": 54}
{"loss": 0.22592370212078094, "acc": 0.9147354364395142, "length": 3.967312812805176, "mode": "train", "epoch": 55}
{"loss": 0.39163118600845337, "acc": 0.8580230474472046, "length": 3.98828125, "mode": "test", "epoch": 55}
{"loss": 0.22477112710475922, "acc": 0.9162244200706482, "length": 3.9792537689208984, "mode": "train", "epoch": 56}
{"loss": 0.37237051129341125, "acc": 0.8663725256919861, "length": 3.99609375, "mode": "test", "epoch": 56}
{"loss": 0.21916644275188446, "acc": 0.9195332527160645, "length": 3.9840850830078125, "mode": "train", "epoch": 57}
{"loss": 0.3938370943069458, "acc": 0.8662108778953552, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2192022055387497, "acc": 0.9195677638053894, "length": 3.985433578491211, "mode": "train", "epoch": 58}
{"loss": 0.3994642496109009, "acc": 0.8599289655685425, "length": 3.99609375, "mode": "test", "epoch": 58}
{"loss": 0.22169220447540283, "acc": 0.9175782203674316, "length": 3.9868106842041016, "mode": "train", "epoch": 59}
{"loss": 0.384945809841156, "acc": 0.8646483421325684, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.2089783251285553, "acc": 0.9240049123764038, "length": 3.985630512237549, "mode": "train", "epoch": 60}
{"loss": 0.39255431294441223, "acc": 0.8626952767372131, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7968749403953552
